### DISCUSSION TITLE
html2canvas doesn't work with servo

### DISCUSSION THREAD

**Discussion by artificial-jon** [servo/servo#27495] [General] [answered]
Here's a simple [codepen](https://codepen.io/art-jamesj/pen/gOrObqE) to show the issue.  It works fine in chrome, but not in servo on macos or hololens.

Note, to make codepen draw ok on servo, Change View -> Full Page View.  Perhaps this is another issue.  :)

**Comment by jdm** [comment, marked as answer]
@artificial-jon I'm going to recreate this as an issue, since it's a pretty exemplary one.

**Reply by artificial-jon** [reply]
Sounds good.  Thanks!

**Comment by CYBAI** [comment]
The filed issue is at https://github.com/servo/servo/issues/27496
